indico/modules/events/editing/controllers/frontend.py

```diff
@@ -31,3 +31,8 @@ def _process(self):
             editable=self.editable,
             contribution=self.contrib
         )
+
+
+class RHEditingTags(RHEditingBase):
+    def _process(self):
+        return WPEditing.render_template('tags.html', self.event)
```

Comment:
What about `management/tags.html` (and `management/editing.html` in `RHEditingDashboard`)?